src/utils/check-clickable-element-in-bubble/check-clickable-element-in-bubble.ts

```diff
@@ -0,0 +1,20 @@
+export const checkClickableElementInBubble = (
+  e: React.SyntheticEvent<Element, Event> | undefined,
+): boolean => {
```

Comment:
```suggestion
export const isInteractiveElement<E extends Element> = (element: E,): boolean => {
```
Я бы проверял сам эелемент. Так и протестить проще будет, и использовать гибче

Reply:
target может быть разным из-за того что в кнопку вложена иконка или ещё что-то

Reply:
можно использовать `curentTarget`

Reply:
обработчик висит не на кнопке, а на блоке

Reply:
на каком блоке? в ПРе только кнопки вижу

Reply:
а, понял, на строке таблицы

Reply:
https://github.com/gpn-prototypes/vega-sp/blob/release/src/pages/projects/ProjectsTable/ProjectsTable.tsx#L183